nixos/lib/make-ext4-fs.nix

```diff
@@ -38,47 +39,11 @@ pkgs.stdenv.mkDerivation {
       truncate -s $bytes $out
       faketime -f "1970-01-01 00:00:01" mkfs.ext4 -L ${volumeLabel} -U ${uuid} $out
 
-      # Populate the image contents by piping a bunch of commands to the `debugfs` tool from e2fsprogs.
-      # For example, to copy /nix/store/abcd...efg-coreutils-8.23/bin/sleep:
-      #   cd /nix/store/abcd...efg-coreutils-8.23/bin
-      #   write /nix/store/abcd...efg-coreutils-8.23/bin/sleep sleep
-      #   sif sleep mode 040555
-      #   sif sleep gid 30000
-      # In particular, debugfs doesn't handle absolute target paths; you have to 'cd' in the virtual
-      # filesystem first. Likewise the intermediate directories must already exist (using `find`
-      # handles that for us). And when setting the file's permissions, the inode type flags (__S_IFDIR,
-      # __S_IFREG) need to be set as well.
       (
         echo write nix-path-registration nix-path-registration
         echo mkdir nix
         echo cd /nix
         echo mkdir store
```

Comment:
Can we also get rid of that glorified `cp ${sdClosureInfo}/registration [img]/nix-path-registration` and `mkdir -p [img]/nix/store` using lkl?

If so, it would remove the need to go through `debugfs` altogether, which would be even better.